src/main.c

```diff
@@ -141,11 +144,11 @@ static void parse_arguments(int argc, char *argv[])
 				g_provided_stream = strdup(optarg);
 				break;
 			case 'd':
-				if (strcmp(optarg, "easy") == 0)
+				if (strcmp(optarg, _("easy")) == 0)
```

Comment:
here you make `easy` which is actually the value of a parameter, a translatable string. but in the description about the values in line 89 you leave it as `easy` thus the two might not match up when people translate into their language differently.
actually i think the parameters should have the same values in all the languages. so lets use `easy` here and not translate it.